content/site-map.yaml

```diff
@@ -36,17 +36,6 @@
 #    n.b. If "file" is an empty string, the navigation item is a "toggle" button where the partial will be omitted from any children navigationItems paths.
 # - "pathOverride" overrides and replaces any built paths from partials.
 
-## Datasets
```

Comment:
Removing datasets from the site-map for both NCPI and AnVIL, as well as any old roadmap references.